v1/linkml-schemas/address.yaml

```diff
@@ -36,15 +37,29 @@ classes:
 
 slots:
   uprn:
+    range: integer
+    slot_uri: adb:uprn
   organisationName:
+    slot_uri: adb:organisationName
   departmentName:
+    slot_uri: adb:departmentName
   subBuildingName:
+    slot_uri: adb:subBuildingName
   buildingNumber:
+    slot_uri: adb:buildingNumber
```

Comment:
buildingNumber is defined as an integer in the addressbase schema, but we seem to be using as a string, so I believe to change this would cause a breaking change.  We should maybe review the slot_uri property if we are not mapping directly to the same type, or we should provide an explanation that we are using the string representation of what is defined in addressbase.

Reply:
Yep, not sure the right thing to do, but we could always record it as an issue that could be resolved in a "version 2" one day. I think maybe slot_uri referencing ADB is fine without having the same exact schema, it's more about whether the semantics are the same - but either way a clarification note against the field would be a good idea.